frontend/TvCv19.Frontend/ClientApp/projects/shared-caregiver/src/lib/nurses-doctors-menu/nurses-doctors-menu.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { ToolbarService } from 'src/app/toolbar.service';
+import { ActivatedRoute } from '@angular/router';
+import {
+  PatientModel,
+  PhysicianModel,
+  CaregiverRouteDataModel,
+  HierarchyLevel,
+} from 'projects/shared/src/public-api';
+
+@Component({
+  selector: 'lib-nurses-doctors-menu',
+  templateUrl: './nurses-doctors-menu.component.html',
+  styleUrls: ['./nurses-doctors-menu.component.scss'],
+})
+export class NursesDoctorsMenuComponent implements OnInit {
+  patients: Array<PatientModel>;
+  careTeam: Array<PhysicianModel> = [];
```

Comment:
is this needed

Reply:
No longer needed, and modified per last comment